scanpy/plotting/_tools/scatterplots.py

```diff
@@ -262,6 +262,7 @@ def embedding(
     #     color=gene2, components = [1, 2], color=gene2, components=[2,3],
     # ]
     for count, (value_to_plot, dims) in enumerate(zip(color, dimensions)):
+        kwargs = kwargs_global.copy()  # is potentially mutated for each plot
```

Comment:
Not a huge fan of `kwargs_global`.  Why not `kwargs_scatter = kwargs.copy()`?  The copy of `kwargs` are only used for the scatter plot and `kwargs_outline` is formed from this copy.